src/gax-internal/src/observability.rs

```diff
@@ -23,10 +23,13 @@ pub mod attributes;
 #[cfg(google_cloud_unstable_tracing)]
 mod errors;
 
-#[cfg(google_cloud_unstable_tracing)]
+#[cfg(all(google_cloud_unstable_tracing, feature = "_internal-http-client"))]
 mod http_tracing;
 
-#[cfg(google_cloud_unstable_tracing)]
+#[cfg(all(google_cloud_unstable_tracing, feature = "_internal-http-client"))]
 pub(crate) use http_tracing::{
     create_http_attempt_span, record_http_response_attributes, record_intermediate_client_request,
 };
+
+#[cfg(all(google_cloud_unstable_tracing, feature = "_internal-grpc-client"))]
+pub mod grpc_tracing;
```

Comment:
I think you need to change `lib.rs` too:

https://github.com/googleapis/google-cloud-rust/blob/467f409f3f3a2283cb527ed0e902b96e7cd9b057/src/gax-internal/src/lib.rs#L39-L40

Otherwise only clients that turn on `_internal-http-client` *and* `_internal-grpc-client` will get this new thing.